core/controllers/reader.py

```diff
@@ -55,6 +55,21 @@
 def require_playable(handler):
     """Decorator that checks if the user can play the given exploration."""
     def test_can_play(self, exploration_id, **kwargs):
+        """Tests if the user can play the exploration.
+        Args:
+            exploration_id: str. The id of the exploration to test if
+                the user can play.
+            kwargs: Any other keyword args.
+
+        Returns:
+            handler or None. If the exploration is disabled,
+            the disabled_exploration.html page is rendered.
+            Otherwise, if the exploration can be played by the user,
+            the corresponding handler is returned.
```

Comment:
I'm not quite sure what a "handler" is; could you elaborate on that? (i.e. what is the handler handling? what's the type?)